R/weights_mcmc_lib.R

```diff
@@ -12,12 +12,16 @@ library(brms)
 #' @param pop_df The population dataframe
 #' @param pop_w Optional.  The weight given to each row of pop_df.  Defaults to ones.
 #' @param save_preds Optional.  If true, save the posterior predictions for re-use.
+#' @param re_formula Optional.  Formula containing group-level effects to be considered in the prediction. If `NULL` (default), include all group-level effects; if `NA`, include no group-level effects.
+#' @param allow_new_levels Optional.  If true, allow new levels of group-level effects in prediction stage.
 #'
 #' @return Draws from the MrP estimate, and the weight vector
 #' whose n-th entry is d E[MrP | X, Y] / d y_n.
 #'
 #'@export
-GetLogitMCMCWeights <- function(logit_post, survey_df, pop_df, pop_w=NULL, save_preds=FALSE) {
+GetLogitMCMCWeights <- function(logit_post, survey_df, pop_df, pop_w=NULL, 
+                                save_preds=FALSE, re_formula=NULL,
+                                allow_new_levels=FALSE) {
```

Comment:
I'll approve the PR, but could you please put in a unit test for these arguments?  I'll make an issue and assign it to you.  That's a good way to make sure we don't break things, especially in the coming refactor.